engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/size/SizeValidatorForCollection.java

```diff
@@ -20,7 +20,7 @@
  *
  * @author Hardy Ferentschik
  */
-public class SizeValidatorForCollection implements ConstraintValidator<Size, Collection<?>> {
+public class SizeValidatorForCollection implements ConstraintValidator<Size, Collection> {
```

Comment:
Having a `SuppressWarnings` and a comment explaining why we did it like that would be nice.

It's not possible to keep the wildcard and fix it in another place?